src/redux/category/categoryIndex.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { Search, selectAllJobs } from '../search/searchSlice';
+import Spinners from '../../components/spinners';
+
+const CategoryIndex = () => {
```

Comment:
- [ ] As required, I kindly suggest that you add a "Back" button on your details page so that it's possible to navigate back to the home page from the details page.

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="946" alt="image" src="https://user-images.githubusercontent.com/52934128/194122696-f437d263-1571-4bec-a081-f34fa7aa5ac4.png">  | <img width="200" alt="image" src="https://user-images.githubusercontent.com/52934128/194127604-024f9180-7e1c-4b83-8f7e-23e822f695cf.png">  |

Reply:
Noted and Implemented!